src/docs/sphinx/user_guide/command_line_options.rst

```diff
@@ -0,0 +1,44 @@
+.. ## Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+.. ## other Serac Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+====================
+Command Line Options
+====================
+
+Below is the documentation for Serac's command line options:
+
+.. list-table:: Options
+   :widths: 25 25 25 25
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Long form
+     - Short form
+     - Variable Type
+     - Description
+   * - --help
+     - -h
+     - N/A
+     - Print this help message and exit
+   * - --input-file
+     - -i
+     - Path
+     - Input file to use
+   * - --restart-cycle
```

Comment:
These are rendering as long dashes in html. Is there a way to escape that?

Reply:
curses! no good deed goes unpunished....

Reply:
fixed by turning off smart quotes on sphinx.